src/app/clients/clients-view/clients-view.component.html

```diff
@@ -29,20 +29,23 @@
         <mat-card-title>
           <h3>
             <i class="fa fa-stop" [ngClass]="clientViewData.status.code|statusLookup" [matTooltip]="clientViewData.status.value"></i>
-            Client Name : {{clientViewData.displayName}}
+            <b>Client Name :</b> {{clientViewData.displayName}}
           </h3>
         </mat-card-title>
 
         <mat-card-subtitle>
           <p>
-            Client #:{{clientViewData.accountNo}} | External
-            Id: {{clientViewData.externalId}} | Staff:{{clientViewData.staffName || 'Unassigned'}}<br />
-            Activation Date : {{(clientViewData.activationDate) ? (clientViewData.activationDate | dateFormat) :'Not Activated'}}<br />
-            Member Of :
-            <span *ngFor="let group of clientViewData.groups">
-              <a>{{group.name}}</a>&nbsp;
+            <b>Office :</b> {{clientViewData.officeName}}<br />
+            <b>Client :</b> {{clientViewData.accountNo}}<br />
+            <b>External Id:</b> {{clientViewData.externalId | externalIdentifier}}<br />
```

Comment:
What is the value of the "externalIdentifier"?

Reply:
The `externalIdentifier` is a new `Pipe` mainly to format (or cut) the long UUID auto generated and just present the latest part.
- If the externalId value is null then will return null, 
- If the externalId value has not an UUID format, then return the same value, and
- If the externalId value has an UUID format, then It is spliced and  return the latest part

And It is a `Pipe` because the externalId value is used in different parts of the UI